contracts/Fees.sol

```diff
@@ -43,7 +43,12 @@ abstract contract Fees is IFees, Owned {
                 protocolWithdrawFee = uint16(updatedProtocolFees & 0xFFF);
 
                 protocolFees = updatedProtocolFees;
-            } catch {}
+            } catch {
+                // note if the ProtocolFeeController reverts we default all protocol fees to 0
+                protocolSwapFee = 0;
+                protocolWithdrawFee = 0;
+                protocolFees = 0;
```

Comment:
I dont think this actually fixes anything.. if the call reverts these values will already be 0 because they wouldn't have gotten reassigned above

Reply:
If the call reverts it will revert `initialize` though which is what we wanted to prevent right